docs/conf.py

```diff
@@ -0,0 +1,160 @@
+# -*- coding: utf-8 -*-
+
+import os
+import sys
+sys.path.insert(0, os.path.abspath('..'))
+
+# -- General configuration ------------------------------------------------
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = [
+    'sphinx.ext.autodoc',
+    'sphinx.ext.intersphinx',
+    'sphinx.ext.napoleon',
+    'sphinx.ext.todo',
+]
+
+# TODO: Please Read!
+# Uncomment the below if you use native CircuitPython modules such as
+# digitalio, micropython and busio. List the modules you use. Without it, the
+# autodoc module docs will fail to generate with a warning.
+# autodoc_mock_imports = ["digitalio", "busio"]
```

Comment:
Surprised Shpinx passed without mocking `digitalio` and `micropython`. :man_shrugging:
 Everything else looks good to me, but I'll leave this as a comment and not a review, for now.

Reply:
I too am surprised by this.

Reply:
oops i dont know. we remove that for pypi anyways so it would be removed anyhow?

Reply:
`digitalio` and `micropython` are brought in via Blinka and the updated requirements.txt. Good job all! 🌮